src/cpp/desktop/DesktopMainWindow.cpp

```diff
@@ -234,6 +234,7 @@ void MainWindow::onWorkbenchInitialized()
 
       avoidMoveCursorIfNecessary();
    });
+   webView()->setFocus();
```

Comment:
Could it be possible for the user to explicitly focus something in the IDE frame before this function is called (before the workbench is initialized)?

I wonder if it would be worth being extra cautious and only doing this if we detect that the main window has focus, but nothing else in the IDE frame.

Reply:
Not sure it's possible, but if it is, where might user put focus outside the web view such that this call would potentially yank it back into the web view? Main menus, I suppose?

Reply:
That's what I was thinking -- it's unlikely, but there might be a case where the user opens a menu but loses focus after the workbench finishes initialization. Probably too unlikely to worth worrying about, though...

Reply:
Famous last words: "Too unlikely to worth worrying about".

I can repro this with 1.2.1578 on Mac (prior to my change):

- launch app
- quickly get focus onto the not-yet-fully-built RStudio menu (with mouse or standard Ctrl+F2 shortcut on a "real" keyboard, doesn't seem to always work on touchbar)
- wait until the UI appears and renders
- keyboard focus is no longer on the menu

So I don't think I'm making it any worse here. Haven't tried Windows, though, so maybe prudent to scope this down to Mac since that's where the problem is.